domains/certificates/Query.API/API/Query.API/ApiModels/Requests/CreateContractValidator.cs

```diff
@@ -10,11 +11,19 @@ public CreateContractValidator()
     {
         var now = DateTimeOffset.UtcNow;
         var utcMidnight = now.Subtract(now.TimeOfDay).ToUnixTimeSeconds();
+        var utcMidnightNextDay = now.AddDays(1).Subtract(now.TimeOfDay).ToUnixTimeSeconds();
 
         RuleFor(cs => cs.StartDate)
             .GreaterThanOrEqualTo(_ => utcMidnight)
             .LessThan(253402300800).WithMessage("{PropertyName} must be before 253402300800 (10000-01-01T00:00:00+00:00)");
 
+        RuleFor(cs => cs.EndDate)
+            .GreaterThanOrEqualTo(cs => cs.StartDate)
```

Comment:
Please change this to be "GreaterThan" as we want at least a non-zero period for the contract